parm/atm/obs/config/sondes.yaml

```diff
@@ -3,7 +3,7 @@ obs space:
   obsdatain:
     obsfile: $(OBS_DIR)/$(OBS_PREFIX).sondes.$(OBS_DATE).nc4
     obsgrouping:
-      group variables: ["station_id", "LaunchTime"]
+      group variables: ["station_id"]
```

Comment:
Removing `LaunchTime` is only necessary because this variable is not in the R2D2 obs.  If we regenerate the R2D2 obs with this variable, we should add `LaunchTime` back.  Is this correct?   If "yes", what's the long term plan for sonde data?  Will it include `LanuchTime`?   If this is where we are heading, should we redo the R2D2 database to include this varaible now?

Reply:
Yes, this is because currently obs come from GSI ncdiags. The operational ncdiags do not contain `LaunchTime`. To get it from ncdiags, we would have to rerun the GSI observer with a modified GSI executable. Long term, this should be coming directly from BUFR files, which should have (hopefully) `LaunchTime`.

Reply:
Got it.  It would be nice to directly read from BUFR files sooner rather than later.   Populating R2D2 with GSI ncdiags is extra work and consumes additional disk space.